tests/test_config_parsing/config_dict_generator.py

```diff
@@ -30,18 +31,39 @@ def define_keys(draw):
 
 
 def touch(filename):
-    with open(filename, "w") as fh:
+    with open(file=filename, mode="w", encoding="utf-8") as fh:
         fh.write(" ")
 
 
 file_names = words
+format_file_names = st.builds(lambda file_name: file_name + "-%d", file_names)
 
 
 @st.composite
 def directory_names(draw):
     return "dir" + draw(words)
 
 
+functions = st.sampled_from(
```

Comment:
Perhaps call these `transforms` instead of `functions` as `functions` sounds like you are making python functions.

Reply:
I have no objections to this. :+1: